week2/homework/src/index.js

```diff
@@ -1,3 +1,79 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+const { readFile, writeFile } = require('fs');
+const { promisify } = require('util');
+
+const reading = promisify(readFile);
+const writing = promisify(writeFile);
+const toDO = 'todolist.json';
+
+
+async function content() {
+    const [, , command, ...args] = process.argv;
+    const entry = args.join(' ');
+    switch (command) {
+        case ('add'): {
+            const currentList = await toRead();
+            currentList.push(entry);
+            if (!entry) { console.log('Please add todo after add command'); return args; }
+            await toWrite(currentList);
+            console.log(entry + ' activity has been added to list  ');
+            break;
+        }
+
+        case ('list'): {
+            const data = await toRead();
+            console.log('Your to do list is : ' + data);
+            break;
+        }
+        case ('reset'): {
+            await toWrite('[]');
+            const data = await toRead();
+            console.log('To do list is empty : ' + data);
+            break;
+        }
+        case ('remove'): {
+            const currentList = await toRead();
+            const remove = new Number(entry) - 1;
+            if (!entry) { return console.log('Please add the number of todo needs to be removed') }
```

Comment:
Same thing here: you start processing the argument before checking if you even should! 😛